src/index.html

```diff
@@ -11,8 +14,71 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <header class="header"></header>
+
+    <main class="main">
+      <section
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__product-image"></div>
```

Comment:
Need to add image with the img tag

Reply:
> Need to add image with the img tag

What class can I add in img tag? I don't know what will be better